src/react-apps/applications/ux-editor/src/components/rightDrawerMenu/CollapsableMenuComponent.tsx

```diff
@@ -0,0 +1,137 @@
+import { createStyles, List, ListItem, ListItemIcon, withStyles } from '@material-ui/core';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import altinnTheme from '../../../../shared/src/theme/altinnStudioTheme';
+
+const styles = createStyles({
+  collapseHeader: {
+    margin: '0 !important',
+    padding: '0 !important',
+  },
+  icon: {
+    padding: '0 0.6rem',
+    width: '2.5rem !important',
+    fontSize: '3rem',
+  },
+  link: {
+    textDecoration: 'underline',
+    textDecorationColor: altinnTheme.altinnPalette.primary.blueDark,
+    cursor: 'pointer',
+  },
+  list: {
+    padding: 0,
+  },
+  listItemHeader: {
+    padding: '1.2rem 0',
+    borderTop: '1px solid ' + altinnTheme.altinnPalette.primary.greyMedium,
+  },
+  listItem: {
+    width: '100%',
+    color: altinnTheme.altinnPalette.primary.blueDarker,
+    fontSize: '1.6rem',
+  },
+  rotateDown: {
+    transform: 'rotate(90deg)',
+    fontSize: '1.3rem',
+    margin: '0 !important',
+    cursor: 'pointer',
+  },
+  rotateRight: {
+    fontSize: '1.3rem',
+    margin: '0 !important',
+    cursor: 'pointer',
+  },
+});
+export interface ICollapsableMenuProvidedProps {
+  classes: any;
+  header: string;
+  listItems: ICollapsableMenuListItem[];
+  menuIsOpen: boolean;
+}
+
+export interface ICollapsableMenuProps extends ICollapsableMenuProvidedProps {
+  language: any;
+}
+
+export interface ICollapsableMenuState {
+  showContent: boolean;
+  menuIsOpen: boolean;
+}
+
+export interface ICollapsableMenuListItem {
+  name: string;
+  action?: () => void;
+}
+
+class CollapsableMenu extends React.Component<ICollapsableMenuProps, ICollapsableMenuState> {
+  constructor(_props: ICollapsableMenuProps, _state: ICollapsableMenuState) {
```

Comment:
I recently learned that the _state parameter is not state, but the context. So no need to have it here.
Doesn't do anything though.